.github/workflows/ci-istio.yml

```diff
@@ -84,14 +84,10 @@ jobs:
           export QUARKUS_CONTAINER_IMAGE_REGISTRY=quay.io
           ./mvnw ${MAVEN_ARGS} clean install -pl :quarkus-integration-test-istio-invoker -De2e-tests -Dkubernetes-e2e-tests
       - name: Report status
-        if: "always() && github.repository == 'quarkusio/quarkus'"
-        shell: bash
-        run: |
-          curl -Ls https://sh.jbang.dev | bash -s - app setup
-          ~/.jbang/bin/jbang .github/NativeBuildReport.java \
-            issueNumber=29536 \
-            runId=${{ github.run_id }} \
-            status=${{ job.status }} \
-            token=${{ secrets.GITHUB_API_TOKEN }} \
-            issueRepo=${{ github.repository }} \
-            thisRepo=${{ github.repository }}
+        if: always() && github.repository == 'quarkusio/quarkus'
```

Comment:
Do we still need the `always()`?

Reply:
Yes. Otherwise it's not executed if a previous step fails.